src/value.rs

```diff
@@ -93,6 +93,7 @@ use gobject_ffi;
 /// (e.g. numeric types) don't.
 ///
 /// See the [module documentation](index.html) for more details.
+#[repr(C)]
 pub struct Value(gobject_ffi::GValue);
```

Comment:
Let's get this in too then, this is not a new requirement for this PR but already before Closure and other places casted between glib::Value and gobject_ffi::GValue for preventing copies.

Reply:
IMHO better be `repr(internal)` but now I can't find it at all.